src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -1579,9 +1582,16 @@ internal void WriteEntityContainer(IEdmEntityContainer container, string fullNam
                 edmNavigationSourceList.Add(singleton);
             }
         }
-
-        this.WriteGeneratedEdmModel(Utils.SerializeToString(this.context.Edmx).Replace("\"", "\"\""));
         
+            if(useTempFile) 

```

Comment:
The indentation on this if\else block is off

Reply:
fixed